### ISSUE TITLE
Sink Connector throughput drops permanently on task restart

### ISSUE THREAD

**Issue by jmwilli25** [confluentinc/kafka-connect-elasticsearch#352]
Environment Details:
Elasticsearch 6.8 (10 data nodes, 3 master)
Elasticsearch connector version (2.2.1)
Kafka connect version (2.2.1)
Kafka broker version (2.2.1)

I have a topic with 7 million records (3 partitions) and deploy an Elasticsearch sink with 1 task using mostly the default configurations. The sink starts by creating the index in Elasticsearch and then starts writing at a rate of 10,000 msgs/second. If I make any changes to the connector's tasks

* pause the connector, restart the task, start the connector
* leave connector running but restart the task

the throughput drops to 400 msgs/second and never recovers to the original 10,000/sec.

If I stop the connector, delete the index from Elasticsearch and resume the connector it goes back to sinking 10k messages/sec.

My connector config:
```
connector.class=io.confluent.connect.elasticsearch.ElasticsearchSinkConnector
type.name=logdata
errors.log.include.messages=true
tasks.max=1
topics=d8.qa.identifi.log.singlesignon.transformed.0
key.ignore=true
schema.ignore=true
value.converter.schemas.enable=false
elastic.security.protocol=PLAINTEXT
name=elasticsearch-sink-d8.qa.identifi.log.transformed.15
connection.url=http://172.30.2.23:9200,http://172.30.0.158:9200,http://172.30.1.63:9200
client.id=elasticsearch-sink-d8.qa.identifi.log.transformed.15
```

NOTE:
This also happens with ES 7.2 and Elasticsearch connector version 2.3.1

**Comment by levzem** [comment]
Hi @jmwilli25, unfortunately this is a known issue with ElasticSearch. Fortunately these two PR's #340 and #309 discuss this issue and resolve it. The fix for this will be released with CP v5.4.

Please let me know if you have further questions, otherwise I will close this issue.

**Comment by jmwilli25** [comment]
Thank you!
